### DISCUSSION TITLE
Lack of parameters for postgresql image.

### DISCUSSION THREAD

**Discussion by huxuan** [apache/airflow#20050] [General]
### Official Helm Chart version

1.3.0 (latest released)

### Apache Airflow version

2.2.1

### Kubernetes Version

Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7", GitCommit:"1dd5338295409edcfff11505e7bb246f0d325d15", GitTreeState:"clean", BuildDate:"2021-01-13T13:15:20Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}

### Helm Chart configuration

N/A

### Docker Image customisations

N/A

### What happened

Can not change the postgresql image.

### What you expected to happen

Be able to change the postgresql image since internal deployment can not download image for dockerhub directly.

### How to reproduce

N/A

### Anything else

Some guidance are needed to submit a PR.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by mik-laj** [comment]
PostgresQL is for development purposes only and for quick testing. We recommend using PostgresSQL installation and configuring the database as external.
Here is docs:  https://airflow.apache.org/docs/helm-chart/stable/production-guide.html

**Comment by mik-laj** [comment]
> Are you willing to submit PR?
>  Yes I am willing to submit a PR!

Feel. free. Assigned.

**Comment by potiuk** [comment]
I still think it's not a good idea to add a change like that - having options would encourage users to use this "not recommended" setting. I think we shoudl really treat the postgres support "as is" just for quick start but having options on that is not a good idea (because we'd have to support all different options there). 

If you want to experiment with different images in your local setting, there is no problem with downloading the chart and replacing it directly in the chart, which gives you a nice development-friendly option, so you are not "left alone" - there is already a possibility of modifying the image for dev settings.

I will convert it into a discussion - maybe there is a good reason and someone would really think it's a good idea, but I think we should not implement it.
